ui/app/src/modules/Preview/Tools/ToolPanel.tsx

```diff
@@ -76,7 +76,7 @@ export function ToolPanel(props: ToolPanelProps): ReactElement | null {
     classes
   } = props;
   return (
-    <div className={classes}>
+    <div className={classes? classes: ''}>
```

Comment:
You can do `classes ?? ''`. 

However, I would think `className={classes}` would be ok as react would handle the nullish value, no? Why did you had to do this change?

Reply:
because React was not handling it

Reply:
fixed